python-menus-toolbars/sample-app.py

```diff
@@ -0,0 +1,238 @@
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Sample PyQt application."""
+
+import sys
+from functools import partial
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QIcon, QKeySequence
+from PyQt5.QtWidgets import (
+    QAction,
+    QApplication,
+    QLabel,
+    QMainWindow,
+    QMenu,
+    QSpinBox,
+    QToolBar,
+)
+
+# NOTE: Uncomment this import to enable icons
+# import qrc_resources
+
+
+class Window(QMainWindow):
+    """Main Window."""
+
+    def __init__(self, parent=None):
+        """Initializer."""
+        super().__init__(parent)
+        self.setWindowTitle("Python Menus & Toolbars")
+        self.resize(400, 200)
+        self.centralWidget = QLabel("Hello, World")
+        self.centralWidget.setAlignment(Qt.AlignHCenter | Qt.AlignVCenter)
+        self.setCentralWidget(self.centralWidget)
+        self._createActions()
+        self._createMenuBar()
+        self._createToolBars()
+        # self._createContextMenu()
```

Comment:
@lpozo That line looks unnecessary until finding the related block of code later, which is also commented out. Can you add a note in a comment here as well to clear things up?

Reply:
I added a comment about that. Let me know if it's okay with you. Thanks!